site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -0,0 +1,271 @@
+We plan to allow the extension to work with old-school 32-bit precision in the future.
+
+While still on the subject of AMD Windows Vulkan drivers, we have to talk about another extension issue.
+Since driver version 22.5.2, support was added for `VK_KHR_push_descriptor`, an old extension that has been working in every other driver for the past 5 years, be it Intel, NVIDIA or Mesa.
+
+While we don't yet know the root cause of the issue, only AMD's Windows drivers crash when calling VK_KHR_push_descriptor. 
+As this extension is critical to the entire rendering process, any AMD GPU would crash on any game.
+
+It seems that this time around, AMD may have simply released a broken implementation of the extension.
+This extension previously worked with yuzu's Vulkan implementation without issue.
+If that’s the case, it’s AMD’s turn to solve the issue.
+In the meantime, toastUnlimited {{< gh-hovercard "8379" "blocked the extension" >}} on the affected AMD Vulkan driver versions.
+
+Elsewhere on the GPU emulation front, [asLody](https://github.com/asLody) {{< gh-hovercard "8311" "implemented stencil fixes when two faces are disabled." >}}
+This should improve rendering for some games that natively use OpenGL.
+
+## HLE Improvements
+
+Moving onto the subject of HLE emulation, a *very dear* section for [bunnei](https://github.com/bunnei). 
+The dev team has been working hard at improving the accuracy and performance of yuzu's kernel emulation.
+
+This time around, a big change was made with how games and the emulated OS can "lock resources". 
+This improves emulation performance with literally every game, and to a varying degree, on any CPU.
+Let's dive in.
+
+In software engineering, a [spinlock](https://en.wikipedia.org/wiki/Spinlock) is a lock that causes a thread trying to acquire it to simply wait in a loop
+ ("spin") while repeatedly checking whether the lock is available.
+
+{{< imgs
+    "./spinlock.png| Example of a spinlock, simple but gets the job done"
+  >}}
+
+There exists another synchronization primitive with a similar function, [the mutex](https://en.wikipedia.org/wiki/Mutual_exclusion). 
+
+The word "mutex" stands for an object providing `MUTual EXclusion` between threads.
+A mutex ensures that only one thread has access to a critical section or data by using operations like a lock and unlock.
+A critical section is a shared resource that many threads want to access.
+While there is no issue if multiple threads want to read the same critical section, no new thread can modify the section until the previous thread finishes its own writing. 
+Under this scenario, the first thread locks the section, and will remain that way until the lock is released.
+
+{{< imgs
+    "./mutex.png| Example of a mutex"
+  >}}
+
+In theory, when a thread tries to lock a mutex and it does not succeed (for example because the mutex is already locked), it will be `paused`.
```

Comment:
I would remove the highlight of paused and sleep; This is not necessarily official terminology, and more of just  a way to explain it in a not overly technical way